CanDIG/playwright-runbook
# Completeness Page UI Test

# Description
This PR adds tests for the CanDIGv2 completeness page, enhancing test coverage.

Key tests implemented:

-  Graph checking of the various graphs on this page
-  Field-level completeness: Ensures that the field-level graph displays percentages as expected

# Types of Change(s)

# How to Run Tests
Make sure to setup your .env file

```
npm install
npx playwright install
npx playwright test ui-tests/completeness-page.ui.ts --workers=1 --ui --headed
```
